db/seeds.rb

```diff
@@ -6,54 +6,80 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-User.create!(email: "email@example.com", password: "password1234")
+User.create!(email: "josh@launchacademy.com", password: "password1234")
+User.create!(email: "anthony@launchacademy.com", password: "password1234")
+User.create!(email: "sean@launchacademy.com", password: "password1234")
+User.create!(email: "aj@launchacademy.com", password: "password1234")
+User.create!(email: "sergei@launchacademy.com", password: "password1234")
 
 CITIES = [
   {
     city: "Boston",
     state: "MA",
-    user_id: User.first.id,
+    user_id: rand(User.all.length)+1,
```

Comment:
you cannot use this to reliably select a random user. It  assumes that all users in the table will have chronological id's - as user's delete their accounts and do other things over time, this will become less consistent. To select a random user, you could use the `RANDOM()` SQL function:

```ruby
User.order("RANDOM()").first
```